src/components/RenderAfterMount/RenderAfterMount.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
Unfortunately, we don't really have a way to test this right now. We'll need to figure out how to have a node environment and jsdom maybe jest [projects](https://jestjs.io/docs/en/configuration#projects-array-string-projectconfig)